<launch>
  <node name="RBFNetwork" pkg="grasp_learning" type="RBFNetwork" output="screen">

    <param name="num_kernels" value="24"/>

   <!-- <param name="num_kernels" value="15"/> --> <!-- Works well for spacing around the manifold  -->

   <param name="num_kernel_rows" value="1"/>

   <!-- <param name="variance" value="0.0001"/> -->
   <param name="variance" value="0.001"/>
   <!-- <param name="variance" value="0.01"/> -->

   <param name="num_policies" value="1"/>
   <param name="num_dim" value="3"/>
   <param name="use_corr_noise" type="bool" value="false"/>
   <param name="burn_in_trials" value="8"/>
   <param name="max_num_samples" value="5"/>
   <param name="relative_path" type="str" value="$(find grasp_learning)/stored_data/data/"/>

   <!-- <param name="spacing_policy" type="str" value="manifold"/> -->

   <param name="spacing_policy" type="str" value="grid"/>

   <param name="grid_x" value="1.0"/>
   <param name="grid_y" value="1.0"/>
   <param name="grid_z" value="1.0"/>

 </node>
</launch>